cmd/juju/backups/restore.go

```diff
@@ -43,6 +53,14 @@ type RestoreAPI interface {
 	RestoreReader(r io.ReadSeeker, meta *params.BackupsMetadataResult, newClient backups.ClientConnection) error
 }
 
+// ModelStatusAPI is used to invoke common.ModelStatus
+//
+//go:generate mockgen -package backups_test -destination modelstatusapi_test.go github.com/juju/juju/cmd/juju/backups ModelStatusAPI
```

Comment:
When keeping mocks in the same package, I used a name like "modelstatusapi_mock_test", so it is still a test file, but also obviously holding a mock(s).